cms/djangoapps/export_course_metadata/signals.py

```diff
@@ -0,0 +1,40 @@
+"""
+This file exports metadata about the course.
+Initially the export goes to S3,
+but the storage class could be used for other backends in the future.
```

Comment:
Also looking at this, why specify here that the export goes to S3? That's an implementation detail and has no impact to this file.